src/renderers/webgl/index.ts

```diff
@@ -30,6 +31,15 @@ export type NodeDragEvent = { type: 'nodeDrag', x: number, y: number, clientX: n
 export type EdgePointerEvent = { type: 'edgePointer', x: number, y: number, clientX: number, clientY: number, target: Graph.Edge, altKey?: boolean, ctrlKey?: boolean, metaKey?: boolean, shiftKey?: boolean }
 
 
+export type AnnotationPointerEvent = { type: 'annotationPointer', x: number, y: number, clientX: number, clientY: number, target: Graph.Annotation, altKey?: boolean, ctrlKey?: boolean, metaKey?: boolean, shiftKey?: boolean }
+
+
+export type AnnotationDragEvent = { type: 'annotationDrag',  x: number, y: number, clientX: number, clientY: number, annotationX: number, annotationY: number,  target: Graph.Annotation, altKey?: boolean, ctrlKey?: boolean, metaKey?: boolean, shiftKey?: boolean }
```

Comment:
what are `annotationX` and `annotationY`? and are they necessary?

Reply:
they are the new x,y coords for the annotation and they take into account the offset of the cursor from the point of origin for the text box. The `x` and `y` are the current coords of the cursor. I mainly followed how we approached this in the node event handlers. Would be open to rethink this though

Reply:
ok, yeah, trying to remember how this works... x/y is the position of the mouse in the graph coordinate system, nodeX/nodeY is the position of the node's center in the graph coordinate system, and clientX/Y is the position of the mouse in the DOM pixel coordinate system.

So yeah, this tracks. tbh I wish I had called nodeX/Y just x/y. but 🤷 .